ecc/bls12381/ff/fp6.go

```diff
@@ -133,10 +137,27 @@ func (z *Fp6) Inv(x *Fp6) {
 	z[1].Mul(c1, t0) // z1 = c1/den
 	z[2].Mul(c2, t0) // z2 = c2/den
 }
+
 func (z *Fp6) Frob(x *Fp6) {
 	z[0].Frob(&x[0])
 	z[1].Frob(&x[1])
 	z[2].Frob(&x[2])
-	z[1].Mul(&z[1], &frob6V1)
-	z[2].Mul(&z[2], &frob6V2)
+	z[1].Mul(&z[1], &Fp2{Fp{}, frob6V1})
+	z[2].Mul(&z[2], &Fp2{frob6V2, Fp{}})
 }
+
+var (
+	// frob6V1 is toMont(v) = 2**384 * v mod fpPrime, where
+	// v = 0x1a0111ea397fe699ec02408663d4de85aa0d857d89759ad4897d29650fb85f9b409427eb4f49fffd8bfd00000000aaac
+	frob6V1 = Fp{fpMont{
```

Comment:
I don't see a test that checks that these are the correct values. Since Frob(x)=x^p, it's not too hard to test against a schoolbook exponentation.

Reply:
new tests were added now.